recipes/virtual_dataframe/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set BASENAME = "virtual_dataframe" %}
+{% if VDF_MODE %}
+  {% set NAME = BASENAME + "-" + VDF_MODE %}
+{% else %}
+  {% set NAME = BASENAME %}
+{% endif %}
+#{% set NAME = BASENAME %}
+{% set PRJ_VERSION = "v0.1.dev69+gaa2a5e5.d20221021" %}
+{% set HASH = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" %}
+{% set PRJ_PACKAGE = BASENAME %}
+{% set PYTHON_VERSION = "3.8" %}
+{% set PYTHON_VERSION_MAX = "3.9" %}
+
+package:
+  name: "{{ NAME }}"
+  version: "{{ PRJ_VERSION }}"
+
+source:
+  # Activate the next line to test the local source code
+#   path: ../../../..
+
+  # Activate url and sha256 to test the published tag
+  # See https://docs.github.com/en/rest/repos/contents#download-a-repository-archive-tar
+    url: https://github.com/pprados/{{ BASENAME }}/tarball/{{ PRJ_VERSION }}/{{ BASENAME }}.tar.gz
+    sha256: {{ HASH }}
+
+build:
+  noarch: python
+  number: 0
+  # no such file directory sur PYTHON ?
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ PRJ_VERSION }}
+  script: python -m pip install --progress-bar off --quiet --no-cache-dir --upgrade --force-reinstall --no-input . -vv
+#   FIXME script: python -m pip install --progress-bar off --quiet --no-input . -vv
+
+requirements:
+  build:
+    - python
+#    - python {{ PYTHON_VERSION }}
+    - pip  # Indispensable pour une installation correct avec scm
+    - setuptools # Indispensable pour une installation correct avec scm
+    - setuptools_scm # Indispensable pour une installation correct avec scm
+
+#  host:
```

Comment:
you almost certainly need a host environment

Reply:
How I can have a host environement ?
I will try without the dependencies of setuptools_scm (I already tested without result).